src/transformers/utils/sentencepiece_model_pb2.py

```diff
@@ -2,7 +2,7 @@
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: sentencepiece_model.proto
```

Comment:
Even if protobuf is normally well thought out for object updates, I still wanted to do some tests to check that this change would not disturb the current behaviour. 

To do this I tokenized the [XNLI dataset](https://huggingface.co/datasets/xnli) with 2 tokenizers - `albert-base-v2` and `microsoft/deberta-v3-large` - by loading the fast version of these tokenizers from the slow version. I did this step twice with the 2 different versions of the `src/transformers/utils/sentencepiece_model_pb2.py` file. At the end I compared the tokenization produced between these 2 versions, and for both tokenizers no changes were identified. I tested with albert which is a tokenizer added in Nov 2019 thus before Dec 2020 and with deberta-v3 because it is known that this tokenizer uses the byte fallback feature and thus was produced with a recent version of sentencepiece.